helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -121,6 +121,11 @@ splunkPlatform:
       enabled: false
       storagePath: "/var/addon/splunk/exporter_queue"
 
+  # Option to disable or enable fsync for filestorage extension used by the agent. Enabling this option will ensure
+  # database integrity at the cost of performance.
```

Comment:
Do you know what the performance cost is by any chance?

Reply:
Not really, performance is mentioned here:
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/storage/filestorage#file-storage

I think that this is strange since our filestorage extension sets fsync to `false` by default (so `nosync` in bbolt is set to `true`), but if we dig more into that, the recommendation is to keep `nosync` flag as `false`. As seen here:
https://pkg.go.dev/go.etcd.io/bbolt#DB

This makes sense as we now know that this flag caused some problems for our customers.
I think that we should expose this flag, keep its value as is (so `fsync` is set to `false`), and maybe do some performance tests to see if we can change it in the future.

Reply:
OK so 2 follow-ups:
* one issue in contrib to have performance test benchmarks with and without this setting
* one issue in contrib to have this setting changed to the right default per bbolt.